<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WIFI</class>
 <widget class="QWidget" name="WIFI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>460</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTreeWidget" name="treeWidgetWiFis">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>566</width>
     <height>416</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:15pt;</string>
   </property>
   <property name="columnCount">
    <number>6</number>
   </property>
   <column>
    <property name="text">
     <string notr="true">NUMBER</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string notr="true">    NAME    </string>
    </property>
   </column>
   <column>
    <property name="text">
     <string notr="true">SECURITY</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string notr="true">RULE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>SUB RULE</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>SIG LEVEL</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="m_button_connect">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>140</y>
     <width>121</width>
     <height>66</height>
    </rect>
   </property>
   <property name="text">
    <string>CONNECT</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>585</x>
     <y>15</y>
     <width>201</width>
     <height>114</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Status</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>WLAN :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1" colspan="2">
     <widget class="QLabel" name="m_label_connectshow">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>ESSID :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" colspan="2">
     <widget class="QLabel" name="m_label_selectSSID">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>PASSWORD :</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="lineedit_password">
      <property name="text">
       <string>jangsungho</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="m_label_state">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="m_button_delete">
   <property name="geometry">
    <rect>
     <x>585</x>
     <y>140</y>
     <width>80</width>
     <height>66</height>
    </rect>
   </property>
   <property name="text">
    <string>START</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_button_stop">
   <property name="geometry">
    <rect>
     <x>585</x>
     <y>215</y>
     <width>80</width>
     <height>66</height>
    </rect>
   </property>
   <property name="text">
    <string>STOP
Pairing</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_button_check">
   <property name="geometry">
    <rect>
     <x>585</x>
     <y>290</y>
     <width>80</width>
     <height>66</height>
    </rect>
   </property>
   <property name="text">
    <string>CHECK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_button_disconnect">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>215</y>
     <width>121</width>
     <height>66</height>
    </rect>
   </property>
   <property name="text">
    <string>DISCONNECT</string>
   </property>
  </widget>
  <zorder>treeWidgetWiFis</zorder>
  <zorder>m_button_connect</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>m_button_delete</zorder>
  <zorder>m_button_check</zorder>
  <zorder>m_button_disconnect</zorder>
  <zorder>m_button_stop</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
